src/coreclr/vm/jitinterface.h

```diff
@@ -352,24 +352,18 @@ extern "C"
 {
 #ifndef FEATURE_EH_FUNCLETS
     void STDCALL JIT_EndCatch();               // JIThelp.asm/JIThelp.s
-#endif // TARGET_X86
+#endif // FEATURE_EH_FUNCLETS
 
     void STDCALL JIT_ByRefWriteBarrier();      // JIThelp.asm/JIThelp.s
 
-#if defined(TARGET_AMD64) || defined(TARGET_ARM)
-
-    FCDECL2VA(void, JIT_TailCall, PCODE copyArgs, PCODE target);
-
-#else // TARGET_AMD64 || TARGET_ARM
-
+#if defined(TARGET_X86) && !defined(UNIX_X86_ABI)
     void STDCALL JIT_TailCall();                    // JIThelp.asm
-
-#endif // TARGET_AMD64 || TARGET_ARM
+#endif // defined(TARGET_X86) && !defined(UNIX_X86_ABI)
 
     void STDMETHODCALLTYPE JIT_ProfilerEnterLeaveTailcallStub(UINT_PTR ProfilerHandle);
 #if !defined(TARGET_ARM64) && !defined(TARGET_LOONGARCH64) && !defined(TARGET_RISCV64)
     void STDCALL JIT_StackProbe();
-#endif // TARGET_ARM64
+#endif // !defined(TARGET_ARM64) && !defined(TARGET_LOONGARCH64) && !defined(TARGET_RISCV64)
```

Comment:
Should the condition here be inverted? Is the function meaningful for RISC platforms?

Reply:
I think that it is. You can link it to https://github.com/dotnet/runtime/issues/13519